docs/spec/abci/client-server.md

```diff
@@ -24,26 +20,34 @@ For each request, a server should respond with the corresponding
 response, where the order of requests is preserved in the order of
 responses.
 
-## Server
+## Server Implementations
 
 To use ABCI in your programming language of choice, there must be a ABCI
-server in that language. Tendermint supports two kinds of implementation
-of the server:
+server in that language. Tendermint supports three implementations of the ABCI, written in Go:
 
-- Asynchronous, raw socket server (Tendermint Socket Protocol, also
-  known as TSP or Teaspoon)
+- In-process (Golang only)
+- ABCI-socket
 - GRPC
 
-Both can be tested using the `abci-cli` by setting the `--abci` flag
+The latter two can be tested using the `abci-cli` by setting the `--abci` flag
 appropriately (ie. to `socket` or `grpc`).
 
+Note that the GRPC version is maintained primarily to simplify onboarding and prototyping and is not receiving the same
```

Comment:
Not sure the spec is the right place for such things.

Reply:
removed